statsmodels/sandbox/nonparametric/densityorthopoly.py

```diff
@@ -337,7 +337,7 @@ def evaluate(self, xeval, order=None):
         xeval = self._transform(xeval)
         if order is None:
             order = len(self.polys)
-        res = sum(c*p(xeval) for c, p in zip(self.coeffs, self.polys)[:order])
+        res = sum(c*p(xeval) for c, p in list(zip(self.coeffs, self.polys))[:order])
```

Comment:
I was asking myself also why I put it there. AFAICS, it is because of the slicing `[:order]`
(I never checked whether it's simple to get only a fixed initial slice of an iterator.
(In this case, it will not make much difference because the order will be a small integer, maybe 3 to 50.)